### ISSUE TITLE
Liman doesn't run behind a proxy

### ISSUE THREAD

**Issue by lukaspj** [salihciftci/liman#13]
**Describe the bug**

Running Liman behind a proxy (e.g. Nginx) mounted to a path (e.g. http://example.com/liman) is not supported because Liman assumes it is mounted to the root, so all the links inside Liman links back to the root.

**Expected behavior**

When Liman is mounted to http://example.com/liman, links on the page should refer to a path relative to the mounted path.

E.g. if Liman is mounted to http://example.com/liman, then clicking on the "containers" menu button should redirect to http://example.com/liman/containers instead of http://example.com/containers.

**Your environment**

Liman version: latest pulled 09 July 2018

**Additional context**

Nginx config:

```conf
location /liman/ {
  proxy_pass http://liman:8080/;
  proxy_redirect off;
  proxy_set_header Host $host:$server_port;
  proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
}
```

**Comment by salihciftci** [comment]
You're right. 

Liman assumes installed in root directory. All links based on that. I will try to fix that.

By the way this is well written bug report. Thank you for that!

**Comment by lukaspj** [comment]
Sounds great!
If it's not trivial to solve "automagically", simply being able to describe the host in an environment variable wouldn't be a bad solution. This is a quite common solution that other services use, in order to work behind a proxy.

However, since Liman doesn't do anything too complex with the URLs, simple relative paths would probably solve the issues.

**Comment by salihciftci** [comment]
https://github.com/salihciftci/liman/commit/34306f824a6825be80152ae463255308f8cd0622

It's in with env variables. It will published with v0.7.
